storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/RocksDbDatabase.java

```diff
@@ -266,12 +277,52 @@ public void storeFinalizedBlocks(final Collection<SignedBeaconBlock> blocks) {
       expectedRoot = block.getParentRoot();
     }
 
+    // Check block signatures are valid for blocks except the genesis block
+    boolean isGenesisBlockIncluded = Iterables.getLast(sorted).getSlot().equals(UInt64.ZERO);
```

Comment:
nit: should use `Constants.GENESIS_SLOT` rather than hardcoding zero here.

Reply:
Done.